src/components/Project/DnDColumnList.tsx

```diff
@@ -124,6 +124,7 @@ export default function DnDColumnList({ projectColumns }: DnDColumnListProps): J
                       ref={provided.innerRef}
                       {...provided.draggableProps}
                       {...provided.dragHandleProps}
+                      tabIndex={-1}
```

Comment:
here it is preventing tab to column component. I know it is not relating to the PR. but it is small enough and I just added anyway.